package.json

```diff
@@ -82,6 +80,10 @@
     "rollup-plugin-uglify": "^2.0.1",
     "styled-components": "^2.2.3"
   },
+  "peerDependencies": {
```

Comment:
These where peer dependencies at one point. I am not sure when they where removed :|
I remember because we had fuzzy versions #84 

Also, we do not need the react-dom peer-dependency. It is not used in the production bundle

Reply:
Done.